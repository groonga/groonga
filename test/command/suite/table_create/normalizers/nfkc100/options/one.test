table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizers 'NormalizerNFKC100("unify_kana", true)'
column_create Terms memos_content COLUMN_INDEX Memos content

load --table Memos
[
{"content": "リンゴ"},
{"content": "りんご"}
]

select Terms --output_columns _key --limit -1
