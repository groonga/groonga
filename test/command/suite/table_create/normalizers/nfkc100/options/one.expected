table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos content COLUMN_SCALAR Text
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizers 'NormalizerNFKC100("unify_kana", true)'
[[0,0.0,0.0],true]
column_create Terms memos_content COLUMN_INDEX Memos content
[[0,0.0,0.0],true]
load --table Memos
[
{"content": "リンゴ"},
{"content": "りんご"}
]
[[0,0.0,0.0],2]
select Terms --output_columns _key --limit -1
[[0,0.0,0.0],[[[3],[["_key","ShortText"]],["ご"],["りん"],["んご"]]]]
